tools/README.md

```diff
@@ -25,7 +25,7 @@ Supple’s tools are categorised so you only need to `@use` the tools you want:
 @use 'node_modules/@supple-kit/supple-css/tools/responsive';
 
 .your-module {
-  @include responsive.mq($from: name) {
+  @include responsive.query('lap') {
```

Comment:
Why? Maybe keep `mq` for Supple's own consistency.

Reply:
True true, I can change that